subprojects/packagefiles/json-c/config/meson.build

```diff
@@ -100,7 +100,14 @@ else
   check_types += 'ssize_t'
 endif
 foreach type : check_types
-  size = cc.sizeof(type, prefix: '#include <stdint.h>')
+  if type == 'size_t'
```

Comment:
This whole section seems like it can be simplified by removing the loop and doing everything manually.

Reply:
Something like

```
conf.set('SIZEOF_INT', cc.sizeof(int))
conf.set('SIZEOF_INT64_T', cc.sizeof(int64_t, prefix: '#include <stdint.h>'))
conf.set('SIZEOF_LONG', cc.sizeof(long))
conf.set('SIZEOF_LONG_LONG', cc.sizeof(long long))
conf.set('SIZEOF_SIZE_T', cc.sizeof(size_t, prefix: '#include <stddef.h>'))
sst = cc.get_argument_syntax() == 'msvc' ? '#include <BaseTsd.h>'  : '#include <sys/types.h>'
conf.set('SIZEOF_SSIZE_T', cc.sizeof('SSIZE_T', prefix: sst))
```

Reply:
I've simplified, but kept the loop.